module/data/item/templates/class.mjs

```diff
@@ -88,6 +89,20 @@ export default class ClassTemplate extends ItemDataModel.mixin(
 
   /* -------------------------------------------- */
 
+  calculateRequiredLp() {
```

Comment:
I dont think this needs an extra function. This is already what the dynamic getter is for. The getter won't likely get more functionality, so this can all be moved into that...

Reply:
true. done